docs/plans/2026-01-17-watcher-start-after-encryption.md

```
@@ -0,0 +1,526 @@
+Emit onboarding-password-set event to frontend
+        │
+        ▼
+Frontend shows success, watcher is now running
+```
+
+---
+
+## Implementation Details
+
+### 1. Port Definition
+
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)